docs/specs/xref.yml

```diff
@@ -42,3 +42,92 @@ outputs:
     { "content": "<p>Link to <a href=\"https://docs.microsoft.com/en-us/dotnet/api/system.string\">String</a></p>\n"}
   build.manifest:
 ---
+# Define uid in conceptual, resolve it
```

Comment:
>Define [](start = 2, length = 6)

can we give an example here for external xref file?

Reply:
That was done in a previous PR, we have a e2e test naming "Restore xref map from configured url and use the map for resolving"